napari/utils/misc.py

```diff
@@ -51,17 +51,26 @@ def parse_version(v) -> 'packaging.version._BaseVersion':
 
 
 def running_as_bundled_app(*, check_conda=True) -> bool:
-    """Infer whether we are running as a briefcase bundle."""
+    """Infer whether we are running as a bundle."""
```

Comment:
Now that we are deprecating this function... I guess I need to revert this?

```suggestion
    """Infer whether we are running as a briefcase bundle."""
```